src/styles/blocks/page.scss

```diff
@@ -0,0 +1,73 @@
+.page {
+  font-family: "Montserrat", sans-serif;
```

Comment:
It's better to use a variable for the font-family to make it easier to maintain the code. Consider adding a variable like $font-family-base and use it here.